yarn.lock

```diff
@@ -12498,6 +12498,14 @@ react-native-vector-icons@^7.1.0:
     prop-types "^15.7.2"
     yargs "^15.0.2"
 
+react-native-vector-icons@^9.1.0:
```

Comment:
Duplicated native dependency. Delete the `react-native-vector-icons@^7.1.0` block above, then add an entry to `resolutions` in `package.json` before re-running yarn. This impacts `edge-login-ui-rn` too.